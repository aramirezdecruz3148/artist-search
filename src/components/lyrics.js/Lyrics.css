@import url('https://fonts.googleapis.com/css?family=Monoton|Poiret+One&display=swap');

.lyricspre{
  font-size: 1.7vw;
  font-family: 'Viga', sans-serif;
  border: solid .3vw #AFEEEE;
  color: rgb(73, 72, 72);
  text-align: center;
  padding: 2vw;
}

.lyricsh3 {
  font-family: 'Monoton', cursive;
  font-size: 3vw;
}