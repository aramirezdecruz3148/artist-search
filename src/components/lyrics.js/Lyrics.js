import React from 'react';
import PropTypes from 'prop-types';

function Lyrics({ title, lyrics }) {
  return (
    <section>
      <h3>{title}</h3>
      <pre>{lyrics}</pre>
    </section>
  );
}

Lyrics.propTypes = {
  title: PropTypes.string.isRequired,
  lyrics: PropTypes.string.isRequired
};

export default Lyrics;
