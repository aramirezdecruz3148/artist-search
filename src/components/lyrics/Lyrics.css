@import url('https://fonts.googleapis.com/css?family=Monoton|Poiret+One&display=swap');

.lyricspre{
  font-size: 1.7vw;
  font-family: 'Viga', sans-serif;
  border: solid .3vw #AFEEEE;
  color: rgb(149, 148, 148);
  text-align: center;
  padding: 2vw;
  overflow: scroll;
}

.lyricsh3 {
  font-family: 'Monoton', cursive;
  font-size: 3vw;
  color: #b7702d;
  text-align: center;
}