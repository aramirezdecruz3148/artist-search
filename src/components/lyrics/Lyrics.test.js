import React from 'react';
import { shallow } from 'enzyme';
import Lyrics from './Lyrics';

describe('Lyrics component', () => {
  it('renders Lyrics', () => {
    const wrapper = shallow(<Lyrics artistName='werw' title='title' lyrics='lyrics' />);
    expect(wrapper).toMatchSnapshot();
  });
});
