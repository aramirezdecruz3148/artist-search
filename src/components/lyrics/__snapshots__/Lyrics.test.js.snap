// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Lyrics component renders Lyrics 1`] = `
<section>
  <h3
    className="lyricsh3"
  >
    Lyrics   for   
    <em>
      title
    </em>
        by  
    werw
  </h3>
  <Nav />
  <pre
    className="lyricspre"
  >
    lyrics
  </pre>
</section>
`;
